examples/query-action/query_action.metatags

```diff
@@ -1,2 +1,2 @@
-description: Example on using Ballerina query action.
-keywords: ballerina by examples, bbe, query, query action, from, join, on, where, let, do, do action, iterable, iterate
+description: Example on performing query actions on iterable items.
```

Comment:
Edit: "Ballerina by Example on performing query actions on iterable items."

or "Example of performing query actions on iterable items."

Follow the same format for all BBEs